pkg/vcdclient/client.go

```diff
@@ -100,6 +105,17 @@ func (client *Client) RefreshBearerToken() error {
 	}
 	client.vdc = vdc
 
+	// reset swagger client
+	swaggerConfig := swaggerClient.NewConfiguration()
+	swaggerConfig.BasePath = fmt.Sprintf("%s/cloudapi", client.vcdAuthConfig.Host)
+	swaggerConfig.AddDefaultHeader("Authorization", authHeader)
+	swaggerConfig.HTTPClient = &http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		},
+	}
+	client.apiClient = swaggerClient.NewAPIClient(swaggerConfig)
+
```

Comment:
Can you change the comment in L94 to something like `reset legacy client`, so that it is clearly evident

Reply:
A log mentioning that we refreshed the clients successfully will be helpful.

Reply:
Done.